ipaserver/plugins/vault.py

```diff
@@ -1117,6 +1150,9 @@ def execute(self, *args, **options):
         # retrieve vault info
         vault = self.api.Command.vault_show(*args, **options)['result']
 
+        wrapping_algo = options.pop('wrapping_algo', None)
```

Comment:
this should be moved before 
```
      vault = self.api.Command.vault_show(*args, **options)['result']
```
or `vault_show()` call would complain with 
```
ipalib.errors.OptionError: Unknown option: wrapping_algo
```

Reply:
good catch, amended.